src/Model/Volunteer.hs

```diff
@@ -0,0 +1,19 @@
+module Model.Volunteer where
+
+import Import
+
+import Control.Monad.Trans.Writer.Strict (tell)
+
+fetchAllInterestsDB :: DB [Entity Interest]
+fetchAllInterestsDB = selectList [] []
+{-# INLINE fetchAllInterestsDB #-}
```

Comment:
Heh, I meant inlining it at the ".hs stage" (aka deleting this function and writing "selectList [] []" instead) :P